images/win/scripts/Installers/Install-AzureModules.ps1

```diff
@@ -36,6 +36,46 @@ foreach ($module in $modules)
         }
     }
 
+    if($null -ne $module.url)
+    {
+        $assets = Invoke-RestMethod $module.url
+    }
+
+    foreach ($version in $module.zipversions)
+    {
+        # Install modules from GH Release
+        if($null -ne $assets) 
+        {
+            $asset = $assets | Where-Object version -like $version `
```

Comment:
why making use of like `version -like $version` shouldn't `-eq` check be more appropriate

Reply:
done